src/main/java/frc/robot/commands/IntakeIn.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Intake;
+
+public class IntakeIn extends Command {
+
+    private Intake intake;
+
+    public IntakeIn(Intake intake){
+        this.intake = intake;
+
+        addRequirements(intake);
+    }
+
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute() {
+        intake.In();
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        intake.stop();
+    }
+
+    @Override
+    public boolean isFinished() {
+        return intake.cheakOut();
```

Comment:
You check that the "note" is outside and that tell you if you succussed in collecting the "note"?

Reply:
you should probably rename it to `hasBall` or something like that. Using a verb isn't appropriate as the method doesn't perform an action really.